index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      rel="stylesheet"
+    />
+
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main-section">
+      <nav>
+        <ul>
+          <li>
+            <div id="logo">
+              <img src="./images/Webp.net-resizeimage.png" alt="" />
+            </div>
+          </li>
+          <li>
+            <span><a href="#">Store</a></span>
+          </li>
+          <li>
+            <span><a href="#">Mac</a></span>
+          </li>
+          <li>
+            <span><a href="#">iPod</a></span>
+          </li>
+          <li>
+            <span><a href="#">iPad</a></span>
+          </li>
+          <li>
+            <span><a href="#">iPhone</a></span>
+          </li>
+          <li>
+            <span><a href="#">iTunes</a></span>
+          </li>
+          <li>
+            <span><a href="#">Support</a></span>
+          </li>
+        </ul>
+        <div class="right-search">
+          <i class="fa fa-search"></i>
+          <input />
```

Comment:
This input should be declared properly with type and other attributes

Reply:
okay fulfilled the input